Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/DocumentPropertyEditor/PrefabComponentAdapter.h

```diff
@@ -31,7 +31,29 @@ namespace AzToolsFramework::Prefab
 
         AZ::Dom::Value HandleMessage(const AZ::DocumentPropertyEditor::AdapterMessage& message) override;
 
+        //! Updates the DPE DOM using the property change information provided. If the property is owned by the focused prefab,
+        //! the change is applied as direct template edit. If the property is owned by descendant of the focused prefab, it is
+        //! applied as an override from the focused prefab.
+        //! @param propertyChangeInfo The object containing information about the property change.
+        void UpdateDomContents(const PropertyChangeInfo& propertyChangeInfo) override;
+
     private:
+
+        //! Creates and applies a component edit prefab patch using the property change information provided.
+        //! @param relativePathFromOwningPrefab The path to the property in the prefab from the owning prefab.
+        //! @param propertyChangeInfo The object containing information about the property change.
+        bool CreateAndApplyComponentEditPatch(
+            AZStd::string_view relativePathFromOwningPrefab,
```

Comment:
Since this is not a public api that other systems will be using, having different signatures is okay imo. Having same signature isn't the ideal case here as the functions do different things.